app/assets/javascripts/email-analytics.js

```diff
@@ -0,0 +1,17 @@
+(function (global, GOVUK) {
+  'use strict'
+
+  window.GOVUK = window.GOVUK || {}
+
+  function canTrack () {
+    return !!GOVUK.analytics
+  }
+
+  // The EmailAnalytics module is a wrapper around GOVUK.analytics
+  GOVUK.EmailAnalytics = {
```

Comment:
I'm not sure if this is needed? Can we just use `GOVUK.analytics.trackEvent` directly? I'm not sure if the wrapper is adding much.